src/styles/note_grid.css

```diff
@@ -0,0 +1,71 @@
+/*Code Copied From https://codepen.io/tobiasahlin/pen/JVmLRa */
```

Comment:
Better to keep CSS file names without underscore, hyphen would be a better choice here.